daffodil-test/src/test/scala/org/apache/daffodil/xml/test/unit/TestDaffodilXMLLoader.scala

```diff
@@ -1,382 +0,0 @@
-   * to a schema supplied by way of the CatalogManager.properties file.
-   *
-   * If the error is detected, then it must have found and used the
-   * user-specified catalog.
-   */
-  @Test def testThisTestRig(): Unit = {
-
-    // LoggingDefaults.setLoggingLevel(LogLevel.Debug)
-
-    val test = new CatalogTestBase {
-
-      override val testCatalog = <catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
-                                   <uri name="foobar" uri={ tmpSchemaFileName.getAbsolutePath }/>
-                                 </catalog>
-
-      // The '5' for line should cause a validation error because it doesn't match
-      // the 'A' enum in the schema.
-      val testData =
-        <onlyAnAWillDo xmlns="foobar">5</onlyAnAWillDo>
-
-      val testSchema =
-        <schema xmlns={ xsd } targetNamespace="foobar" xmlns:xsd={ xsd }>
-          <!-- in our internal schema line is an int, in this schema it is a string.
-             So if our internal schema is being used, a validation against that will cause an error
-             for anything that doesn't obey the syntax of an int. So we want to demonstrate
-             that we get that error.
-          -->
-          <xsd:element name="onlyAnAWillDo">
-            <xsd:simpleType>
-              <xsd:restriction base="xsd:string">
-                <xsd:enumeration value="A"/>
-              </xsd:restriction>
-            </xsd:simpleType>
-          </xsd:element>
-        </schema>
-
-      override def runTest(): Boolean = {
-        if (!super.runTest()) {
-          System.err.println("test skipped.")
-          return false
-        }
-        // but when it has to resolve our dafint namespace, we don't want it to take the 
-        // one we created here, rather we want it to take our internal one.
-        val excs = exceptionList
-
-        println(excs)
-        //
-        // Now, our internal dafint schema requires the line attribute to be a 
-        // xs:int, so using 'A' above means we should get an error on the line attribute.
-        //
-        // these checks pass if we got an error complaining about the 'line'
-        //
-        assertTrue(excs.length >= 1)
-        // check that we got an error message complaining about the element.
-        val hasError = excs.exists { _.getMessage().contains("onlyAnAWillDo") }
-        assertTrue(hasError)
-        true
-      }
-    }
-
-    test.runTest
-  }
-
-  /**
-   * This test insures that a user catalog doesn't get priority over
-   * our internal catalog by defining a catalog and schema that uses
-   * one of our namespaces.
-   *
-   * Our dafint.xsd defines testElement and the line and col attributes
-   * must be type xs:int. The data here has dafint:line with value 'A',
-   * so if the built-in schema is used, it will find this error. So the
-   * test passes if the error is detected.
-   */
-  @Test def testThatUserCatalogCannotOverrideDaffodilInternalCatalog1(): Unit = {
```

Comment:
I think we do need a way to test this, or XML catalogs are just going to stop working at some point. By deleting these tests, instead of commenting them out with // TODO type comments around them, they are out of site/out of mind, and will be forgotten by the time things break. 

I'm fine with removing these tests if we either book a JIRA ticket to add testing of this and include a git hash that still has these tests in it, so they're easily retrieved, OR, just comment these out and put //TODO. Either way.

Reply:
Agreed. I'll create a ticket to add CatalogManager.properties tests, and reference a commit hash that includes these tests as good candidates once we determine a reasonable way to test this.

Reply:
Created [DAFFODIL-2424](https://issues.apache.org/jira/browse/DAFFODIL-2424) to track the issue of adding back tests.